docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ros:noetic-ros-base-focal
+
+RUN apt-get update && \
+    apt-get install -y \
+        apt-utils \
+        build-essential \
+        git \
+        python3-pip \
+        python-is-python3 \
+        python3-catkin-tools
+
+# Use the following command in a running container to find dependencies that we
+# can install ahead of time. We do this to make reinstalling dependencies faster
+#
+#   rosdep install --from-paths src --simulate --reinstall
+RUN apt-get install -y \
+        ros-noetic-catkin \
+        ros-noetic-message-runtime \
+        ros-noetic-message-generation \
+        python3-numpy \
+        ros-noetic-cv-bridge \
+        ros-noetic-ros-pytest
```

Comment:
Overall kind of annoying -- would be nice to parse the output and automatically generate a delimited list of dependencies.

```bash
$ rosdep install --from-paths . --simulate --reinstall
#[apt] Installation commands:
  sudo -H apt-get install ros-noetic-message-runtime
  sudo -H apt-get install python3-numpy
  sudo -H apt-get install ros-noetic-cv-bridge
  sudo -H apt-get install ros-noetic-catkin
  sudo -H apt-get install ros-noetic-ros-pytest
  sudo -H apt-get install ros-noetic-message-generation
```